hrm-service/src/post-survey/post-survey-data-access.ts

```diff
@@ -0,0 +1,32 @@
+import { db } from '../connection-pool';
+import { SELECT_POST_SURVEYS_BY_CONTACT_TASK } from './sql/post-survey-get-sql';
+import { insertPostSurveySql } from './sql/post-survey-insert-sql';
+
+export type NewPostSurvey = {
+  contactTaskId: string;
+  taskId: string;
+  data: Record<string, any>;
+};
+
+export type PostSurvey = NewPostSurvey & {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
```

Comment:
We have some records `createdAt` and `updatedAt` as `string`, others as `Date`. Do you think is worth at some point standardizing them?

Reply:
Probably - at the moment we don't actually do anything with the values except write them out in the response, so the type isn't used.

Still, it's good to standardise to be prepared for the future. Will change these to Date in a follow on refactor PR